models/sfr/pincell/make_openmc_model.py

```diff
@@ -0,0 +1,42 @@
+import openmc
+from numpy import sqrt
+from argparse import ArgumentParser
+from models.sfr import common_input as pincell_params
+import models.sfr.openmc_pincells as pins
+from models.lwr.openmc_settings import COMMON_SETTINGS
```

Comment:
I guess I wasn't clear, but I didn't necessarily mean that you had to use the LWR settings.  It probably makes sense to have your own settings file for now, but it can be identical to the LWR one, and placed in the same relative place in the directory hierarchy